modules/admin_manual/pages/configuration/user/user_management.adoc

```diff
@@ -253,6 +259,15 @@ against quotas.
 When a user creates a public link share via URL, and allows uploads, any
 uploaded files count against that user’s quota.
 
+== User GDPR Requests
+
+To comply with article 15 of the GDPR framework, ownCloud provides buttons in the users personal settings view which can be accessed via menu:User Name[Personal > General > Account] to:
+
+* Request data export
+* Request account deletion
+
+When a user clicks on one of these buttons, the admin group will recieve an email with the users request for further processing.
+
 == Moving the User Home
 
 Usually, a user's home folder is located in the `data/` directory, the location of which is defined in `config.php` and defaults to `/var/www/owncloud/data`. Usually in smaller installations, the data directory mounts to a physical drive of the server and in bigger installations the data directory is mounted via NFS. This can be less than optimal for very big installations with thousands of users and/or users with very big space consumption. This can result in a high load on a single mount point.
```

Comment:
```suggestion
Usually, a user's home folder is located in the `data/` directory, the location of which is defined in `config.php` and defaults to `/var/www/owncloud/data`. In smaller installations, the data directory is often mounted on a physical drive of the server. On larger installations, the data directory is usually mounted via NFS. This can be less than ideal for very big installations with thousands of users and/or users with very big space consumption. This can result in a high load on a single mount point.
```